Model/MenuRepository.php

```diff
@@ -133,6 +142,7 @@ public function getList(SearchCriteriaInterface $criteria)
         $collection->setPageSize($criteria->getPageSize());
         $objects = [];
         foreach ($collection as $objectModel) {
+            $objectModel->setHasDataChanges(false);
```

Comment:
I have a question about this line, I have tested the module, and this line doesn't make any difference (at least during my test cases), is there any reason why was it added? maybe I am missing some case and this is useful to avoid issues?

Reply:
https://github.com/SnowdogApps/magento2-menu/pull/128#discussion_r487841317

Right now, only the [nodes repository collection items has the save issue](https://github.com/SnowdogApps/magento2-menu/blob/837ce8d2ed355bc8e19f0b61e595742b10b503db/Controller/Adminhtml/Menu/Save.php#L201), so I could remove this line from menu repository.

Reply:
Indeed only node repository has the issue, therefore, please remove this line, thanks in advance

Reply:
I removed this one.